pkgs/by-name/so/sourcegit/package.nix

```
@@ -79,6 +79,7 @@ buildDotnetModule (finalAttrs: {
       name = "SourceGit";
       exec = "SourceGit";
       icon = "SourceGit";
+	  categories = ["Development"];
```

```suggestion
      categories = [ "Development" ];
```

https://github.com/NixOS/nixpkgs/actions/runs/18818604731/job/53690593723?pr=455855